### DISCUSSION TITLE
root:~# go install -v github.com/projectdiscovery/naabu/v2/cmd/naabu@latest  go/pkg/mod/github.com/refraction-networking/utls@v1.5.2/cfkem.go:25:2: package crypto/ecdh is not in GOROOT (/usr/lib/go-1.18/src/crypto/ecdh) root@ser230545925848:~#

### DISCUSSION THREAD

**Discussion by N0o01** [projectdiscovery/naabu#806] [Q&A]
root@ser230545925848:~# go install -v github.com/projectdiscovery/naabu/v2/cmd/naabu@latest

go/pkg/mod/github.com/refraction-networking/utls@v1.5.2/cfkem.go:25:2: package crypto/ecdh is not in GOROOT (/usr/lib/go-1.18/src/crypto/ecdh)
root@ser230545925848:~#

**Comment by MetzinAround** [comment]
Hi @N0o01! looks like you may need the latest version of go. I can see in the text above that you are using go 1.18. Most ProjectDiscovery tools require v1.20. 

Let us know if you have any other questions!

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
